content/en/docs/disk-encryption.md

```
@@ -0,0 +1,61 @@
+---
+linkTitle: LINSTOR Encrypted Storage
+title: LINSTOR Encrypted Storage
+description: LINSTOR Encrypted Storage
+weight: 100
+---
+
+
+Create encryption passphrase:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)